src/main.rs

```diff
@@ -20,6 +20,7 @@ use config::{CargoCratesToml, Package, UserConfig};
 fn main() {
     if let Err(err) = wrapped_main() {
         error!("{}", err);
+        std::process::exit(1);
```

Comment:
Considering the function's documentation, I would prefer if this was done through [`ExitCode`](https://doc.rust-lang.org/stable/std/process/struct.ExitCode.html): it would deviate from returning a `Result` only in that it does not `eprintln!` anything, which is what we want here. You can refactor to use a `match` here if you want. Also, just to avoid a potential useless back-and-forth, import the struct instead of using it directly through `std::...`.

Reply:
I have updated the main function - let me know if you had something else in mind. I nearly went with this approach in the initial PR but then decided to go for the simple `process::exit` approach.

Reply:
Thanks.